pycrdt_websocket/yroom.py

```diff
@@ -305,7 +305,11 @@ async def serve(self, websocket: Websocket):
         except Exception as exception:
             self._handle_exception(exception)
 
-    async def local_update_awareness(self, state):
+    def local_update_awareness(self, state: bytes):
+        if self._task_group is not None:
```

Comment:
We should probably raise an exception if it's `None`.

Reply:
Done in https://github.com/jupyter-server/pycrdt-websocket/pull/73/commits/44ce8528172bbda2c631960beac7f0b49f834a71